xml/deploy_cephadm.xml

```diff
@@ -701,131 +701,315 @@ o- time_server ................................................ [enabled]
-      used for the deployment or migration of monitoring components.
-     </para>
-     <para>
-      For more information about the images used by the monitoring stack and
-      how to customize them, visit the
-      <xref linkend="monitoring-custom-images"/> page.
-     </para>
-    </note>
-   </sect3>
    <sect3 xml:id="deploy-cephadm-configure-registry">
-    <title>Configuring the container registry</title>
+    <title>Using the container registry</title>
     <para>
-     Optionally, you can set a local container registry. This will serve as a
-     mirror of the <literal>registry.suse.com</literal> registry. Remember you
-     need to re-sync the local registry whenever there are new updated
-     containers available from
-     <systemitem class="systemname">registry.suse.com</systemitem>.
-<!-- 2021-02-09 tbazant: still looking for better solution
-     Find more
-     information about creating a container registry in
-     <link xlink:href="https://documentation.suse.com/sbp/all/single-html/SBP-Private-Registry/"/>
-     -->
-    </para>
-    <para>
-     Creating a local registry is useful in the following scenarios:
+     The &ceph; cluster needs to have access to a container registry so that it
+     can download and deploy containerized &ceph; services. There are two ways
+     to access the registry:
     </para>
     <itemizedlist>
      <listitem>
       <para>
-       You have a lot of cluster nodes and want to save download time and
-       bandwidth by creating a local mirror of container images.
+       If your cluster can access the default registry at
+       <literal>registry.suse.com</literal> (directly or via proxy), you can
+       point &cephsalt; directly to this URL without creating a local registry.
+       Continue by following the steps in
+       <xref linkend="deploy-cephadm-configure-imagepath"/>.
       </para>
      </listitem>
      <listitem>
       <para>
-       Your cluster has no access to the online registry (an air-gapped
-       deployment) and you need a local mirror to pull the container images
-       from.
+       If your cluster cannot access the default registry&mdash;for example,
+       for an air-gapped deployment&mdash;you need to configure a local
+       container registry. After the local registry is created and configured,
+       you need to point &cephsalt; to it.
       </para>
      </listitem>
-     <listitem>
+    </itemizedlist>
+    <sect4 xml:id="updating-ceph-local-registry">
+     <title>Creating and configuring the local registry (optional)</title>
+     <important>
       <para>
-       If configuration or network issues prevent your cluster from accessing
-       remote registries across a secure link, so you need a local, unencrypted
-       registry instead.
+       There are numerous methods of creating a local registry. The
+       instructions in this section are examples of creating a secure and
+       insecure registry.
```

Comment:
```suggestion
       instructions in this section are examples of creating secure and
       insecure registries.
```

Reply:
Since we are discussing 2 examples of 2 different types of registry, this seems more logical to me than the superficially self-contradictory "secure _and_ insecure". Surely if it were a single registry, it must be one or the other, not both?

Reply:
Sounds like a reasonable change to me